src/components/tools/text-tool.tsx

```diff
@@ -93,6 +94,7 @@ export default class TextToolComponent extends BaseComponent<IToolTileProps, ISt
   private tileContentRect: DOMRectReadOnly;
   private toolbarToolApi: IToolApi | undefined;
   private plugins: HtmlSerializablePlugin[] | undefined;
+  private textOnFocus: any;
```

Comment:
Should be able to type this more precisely.

Reply:
I used any here because `prevText` above was also `any`. But I could update it... I'm just struggling to understand how to get the syntax right... It looks like it is this: https://github.com/concord-consortium/collaborative-learning/blob/master/src/models/tools/text/text-content.ts#L25, but I can't figure out how to get the compiler to be happy with that. If I expand it to VSCode's suggestion, it is... a bit complicated:
![image](https://user-images.githubusercontent.com/15675877/191264770-5feb1e7f-4ddb-40f3-abd4-d56bb5923031.png)

Reply:
Nevermind. It turns out what it really means is string | string [] | undefined which is reasonable :)